templates/c_quiz_start.html

```diff
@@ -5,53 +5,33 @@
     <title>Platform</title>
     <meta name="viewport" content=
         "width=device-width,initial-scale=1">
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/c_quiz.css') }}">
-  
-<!--navigation bar and title-->
+    <link rel="stylesheet" href="c_quiz.css">
   </head>
+  
+  <!--navigation bar and title-->
   <body>
     <div class="background">
-        <div class="nav">
-          <a href="/quiz" class="active"><strong>Take Quiz</strong></a>
-          <a href="/toolkit">View Toolkit</a>
-          <a href="/profile">Your Profile</a>
-          <a href="/home">Home</a>
+      <div class="nav">
+        <div class="profile-dropdown-menu">
+          <a id="navmenu">Your Profile</a>
+          <ul class="profile-dropdown-content">
+            <a href=c_profile.html>Edit Profile</a>
+            <a href=c_login.html>Logout</a>
+          </ul>
         </div>
+        <a href=c_quiz_start.html class="active" id="navmenu"><strong>Take Quiz</strong></a>
+        <a href="c_view.html" id="navmenu">View Toolkit</a>
+        <a href=c_main.html id="navmenu">Home</a>
+      </div>
           <div class="text-box">
             <h2>Quiz: What Do You Know About Unconscious Bias?</h2>
           </div>
           <div class="start-screen">
               <!--<p>Take this quiz to test your understanding!</p>-->
-              <a id="start-button" href="/quiz-question">Start</button>
+              <a id="start-button" href=c_quiz_question.html>Start</a>

```

Comment:
Cannot add hyperlink to HTML file directly. Use a corresponding route defined in program1.py